src/Features/SupportLegacyModels/EloquentModelSynth.php

```diff
@@ -210,7 +210,9 @@ protected function loadModel($meta): ?Model
                 $query->with($meta['relations']);
             }
 
-            $model = $query->first();
+            if ($query->first() !== null) {
```

Comment:
`$query->first();` will already return `null` if nothing is found.

Reply:
Ok, thank you for the suggestion!